ewlegal-utils.sty

```diff
@@ -0,0 +1,35 @@
+%% TODO Stock beginning text, copyrights and project info and all that
+
+\ProvidesPackage{engwales-utils}[2021-02-21 England and Wales Legal Utilities]
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% Variables
+%%%%%
+
+%% Sets optional options of the point env
+% \newcommand{\pointspacing}{leftmargin=0cm,labelsep=1cm}
+% TODO This work around to reusing the point env does not work. FIX!
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% Typeset Elements
+%%%%%
+
+%% Implement tramlines for use in titles, element is not centred, or bolded
+\newcommand{\tramlines}[1]
```

Comment:
can we add a TODO for cleaning up the magic numbers? probably with optional args.

Reply:
I added a brief todo.